extensions/amp-fit-text/0.1/test/test-amp-fit-text.js

```diff
@@ -71,6 +72,25 @@ describes.realWin(
       await ft.implementation_.mutateElement(() => {});
       const content = ft.querySelector('.i-amphtml-fit-text-content');
       expect(content.textContent).to.equal(newText);
+      (await ft.implementation_).unlayoutCallback();
+    });
+
+    it('re-calculates font size if a resize is detected by the measurer', async () => {
+      const ft = await getFitText(
+        'Lorem ipsum dolor sit amet, has nisl nihil convenire et, vim at aeque inermis reprehendunt.'
+      );
+      expect(ft.style.fontSize).to.equal('17px');
+      const setupUpdateFontSizeSpy = env.sandbox.spy(
+        ft.implementation_,
+        'updateFontSize_'
+      );
+      ft.setAttribute('width', '50');
+      ft.setAttribute('height', '100');
+      ft.style.width = '50px';
+      ft.style.height = '100px';
```

Comment:
Do we need both of these? Does this work?
```suggestion
      ft.style.width = '50px';
      ft.style.height = '100px';
```

Reply:
Ah yes! that does work, making the change.